proxmox/client.go

```diff
@@ -256,6 +256,21 @@ func (c *Client) GetVmConfig(vmr *VmRef) (vmConfig map[string]interface{}, err e
 	return c.GetItemConfigMapStringInterface("/nodes/"+vmr.node+"/"+vmr.vmType+"/"+strconv.Itoa(vmr.vmId)+"/config", "vm", "CONFIG")
 }
 
+// GetStorage Get status for all datastores
+func (c *Client) GetStorage(nodeName string) (storageStatus []interface{}, err error) {
+	var data map[string]interface{}
+	url := fmt.Sprintf("/nodes/%s/storage/", nodeName)
+	err = c.GetJsonRetryable(url, &data, 3)
```

Comment:
maybe we should do the "3" - numbers of tries as configurable? 
I mean tomorrow we can think that 5 is better or 2 is better

Reply:
All this library is written like that, I just added another endpoint..
But I will fix it :)